packages/charts/react-charts-preview/library/src/components/DonutChart/DonutChartRTL.test.tsx

```diff
@@ -115,8 +115,8 @@ describe('Donut chart interactions', () => {
 
     //single click on first legend
     fireEvent.click(legend!);
-    const getById = queryAllByAttribute.bind(null, 'id');
-    expect(getById(container, /Pie.*?second/i)[0]).toHaveAttribute('opacity', '0.1');
+    const getByClass = queryAllByAttribute.bind(null, 'class');
+    expect(getByClass(container, /fui-donut-arc__root/i)[1]).toHaveStyle('opacity: 0.1');
```

Comment:
dont use class names. These are internal and can change anytime

Reply:
Ok, I'll switch to id